src/components/HomeLayout/styles.css

```diff
@@ -682,3 +682,66 @@
 p.libraries__feature {
   flex-grow: 1;
 }
+
+.mana_calculator__card {
+  background-color: var(--ifm-color-emphasis-100);
+  border: 1px solid var(--ifm-color-emphasis-200);
+  border-radius: var(--ifm-global-radius);
+  color: inherit;
+  transition: border-color 200ms ease-in-out;
+  padding: 15px;
+  margin: 10px 0px;
+}
+
+.mana_calculator_inner__card {
+  background-color: var(--ifm-color-emphasis-0);
+  box-shadow: none;
+  border: 1px solid var(--ifm-color-emphasis-300);
+  border-radius: var(--ifm-global-radius);
+  color: inherit;
+  transition: border-color 200ms ease-in-out;
+  padding: 15px;
+  margin: 10px 0px;
+}
+
+.mana_calculator__card input:not([role$='combobox']) {
+  background-color: var(--ifm-color-emphasis-200) !important;
+  padding: 10px;
+  border-radius: 4px;
+  border: none;
+}
+
+.input--vertical-spaced {
+  margin-bottom: 1rem;
+}
+
+.horizontal-spaced {
+  margin-right: 5px;
+}
+
+.button--remove {
+  padding: 10px;
+}
+
+.row--centered {
+  display: flex;
+  align-items: center;
+}
```

Comment:
im not a big fan of these atomic classes here, they are merely used in the mana calculator too so i think we could move them there